app/presenters/advanced_search_result_set_presenter.rb

```diff
@@ -4,7 +4,6 @@ class AdvancedSearchResultSetPresenter < ResultSetPresenter
   def to_hash
     super
       .merge(applied_filters: applied_filters_or_all_subgroups)
-        .except(:atom_url)
```

Comment:
Comparing the deployment to live, it doesn't look like this version is updating the atom link in the head anymore...

`<link rel="alternate" type="application/atom+xml" title="ATOM" href="/news-and-communications.atom">`

Is this intentional?